app/Http/Controllers/ContactsController.php

```diff
@@ -240,7 +240,7 @@ public function update(Request $request, Contact $contact)
                 ->withErrors($validator);
         }
 
-        if (! $contact->setName($request->input('firstname'), null, $request->input('lastname'))) {
+        if (! $contact->setName($request->input('firstname'), $request->input('lastname'), null)) {
```

Comment:
We should remove the `null` value here, because the third parameter of `setName` is optionnal.